### ISSUE TITLE
[MU4 Issue] A certain note is accented in a specific score without being notated as an accent

### ISSUE THREAD

**Issue by KaiVinter** [musescore/MuseScore#12607]
**Describe the bug**
See in the attached score and video the issue when playing back. It doesn't matter if you start playback from the beginning, it's just that note gets accented anyways. The score was created in MS4.

[Untitled score.mscz.zip](https://github.com/musescore/MuseScore/files/9223039/Untitled.score.mscz.zip)

**To Reproduce**
1. Open attached score
2. Pless play
3. Hear error

**Expected behavior**
Notes shoudn't be accented if there no accent written.

**Video**

https://user-images.githubusercontent.com/88064903/181839419-b806ad79-7bf8-4c60-9abb-42d7ba71a062.mp4

**Platform information**
 - OS: MacOS Mojave 10.14.6

**Additional context**
Not related, but the order of the instruments in the mixer does not match the order in the score.

**Comment by fredg999** [comment]
I wasn't able to reproduce the bug (OS: Ubuntu 20.04).

However, I have experienced a similar issue with one of my scores as well. In both cases, it's the last of a series of notes with the same rhythm and pitch, but I have no idea whether that's relevant or just a coincidence. I also experienced the issue with Muse Sounds only, not with MS Basic. BTW I won't be uploading my file because it ended up being corrupted at some point (and I'm not sure why either); it still opens and works mostly fine, but I wouldn't try fixing a bug using a corrupted file for reference.

Regarding the order of the instruments in the mixer, it should update if you close and reopen the mixer, but it doesn't seem to update in real time when you drag around the instruments in the instruments panel, like the score does for example.

Another thing that bugs me is that your mixer shows "MS Basic", but the playback sounds like Muse Sounds. I wonder how that happened.
